client/ayon_resolve/plugins/publish/collect_current_project.py

```diff
@@ -1,30 +1,29 @@
 import pyblish.api
 
-from ayon_core.hosts.resolve import api as rapi
-from ayon_core.hosts.resolve.otio import davinci_export
+from ayon_resolve import api
 
 
 class CollectResolveProject(pyblish.api.ContextPlugin):
     """Collect the current Resolve project and current timeline data"""
 
     label = "Collect Project and Current Timeline"
     order = pyblish.api.CollectorOrder - 0.499
+    hosts = ["resolve"]
 
     def process(self, context):
-        resolve_project = rapi.get_current_resolve_project()
+        resolve_project = api.get_current_resolve_project()
         timeline = resolve_project.GetCurrentTimeline()
-        fps = timeline.GetSetting("timelineFrameRate")
-
-        video_tracks = rapi.get_video_track_names()
 
-        # adding otio timeline to context
-        otio_timeline = davinci_export.create_otio_timeline(resolve_project)
+        video_tracks = api.get_video_track_names()
+        otio_timeline = api.export_timeline_otio(timeline)
+        current_file = resolve_project.GetName()
```

Comment:
```suggestion
        host = registered_host()
        current_file = host.get_current_workfile()
```

And also add: 
- `from ayon_core.pipeline import registered_host`

Reply:
this should fix the issue from here: https://github.com/ynput/ayon-resolve/pull/15#issuecomment-2328493576

Reply:
this is updated !